src/main/java/org/reaktivity/specification/nukleus/oauth/internal/OAuthJwtKeys.java

```diff
@@ -80,23 +83,19 @@
         es256.put("y", "x_FEzRu9m36HLN_tue659LNpXW6pCyStikYjKIWI5a0");
         es256.put("d", "jpsQnnGQmL-YBIffH1136cspYG6-0iY7X1fCE9-E9LI");
 
-        KeyPair rsKeyPair = null;
-        KeyPair esKeyPair = null;
         try
         {
-            rsKeyPair = initRSAKeyPair(rsa256);
-            esKeyPair = initECKeyPair(es256);
+            RFC7515_RS256 = initRSAKeyPair(rsa256);
+            RFC7515_ES256 = initECKeyPair(es256);
         }
-        catch(Exception e)
+        catch(Exception ex)
         {
-            // Utility
+            throw new IllegalStateException(ex);
         }
-        RFC7515_RS256 = rsKeyPair;
-        RFC7515_ES256 = esKeyPair;
     }
 
     private static KeyPair initRSAKeyPair(
-        Map<String, String> params) throws Exception
+        Map<String, String> params) throws NoSuchAlgorithmException, InvalidKeySpecException
```

Comment:
Simplify to `GeneralSecurityException` which is a common super type for these exceptions.
Please apply to all similar method signatures.